packages/hw-common/package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "@web3-onboard/hw-common",
+  "version": "1.0.0-alpha.1",
```

```suggestion
  "version": "2.0.0-alpha.1",
```
Since we started all other packages at version 2 I think we can do the same here